guw/main.py

```diff
@@ -208,8 +214,6 @@ def markdown(self):
             li = "* "
             if feature["status"] == "integrated":
                 li += "🟢"
-            elif feature["status"] == "merged":
```

Comment:
Why? IMHO Listing work done is interesting.

![image](https://github.com/user-attachments/assets/10a783f2-fec6-4adc-b7c2-942bed25c408)

Reply:
`merged`is an intermediate state. It only says that the feature is merged upstream but never integrated back. The problem is that if you sync again, from zero, those changes are already in main and then the sync misbehave. Instead of doing two sync processes (one with merged and another with integrated) manually and immediately, just do one and maintain the `merged` but as an internal status.